src/backend/routes/tankRoutes.js

```diff
@@ -59,4 +58,13 @@ router.put('/casusUpdate/:tankId', [
         .exists(),
     ], tankController.casusUpdate);
 
+// Deletes tank from DB
+// Route Call: /deleteTank/<tankId>
+// Req must contain the tankId within the uri of the api call in place of <tankId>
+// Returns confirmation message that tank was deleted.
+router.delete('/deleteTank/:tankId', [
+    check('tankId', 'tankId should be a MongoId string')
+        .isMongoId()
```

Comment:
Forgot that this option exists. I might do another rundown at some time to make sure any relevant strings that are only for MongoId uses this validation instead of isString()